wireshark/USAGE.md

```diff
@@ -1,43 +1,55 @@
-# OQS-wireshark
+This project provides a Docker image to build [Wireshark](https://www.wireshark.org/) with quantum-safe cryptography
+support through the [Open Quantum Safe (OQS) provider](https://github.com/open-quantum-safe/oqs-provider). This Docker
+image allows Wireshark to analyze network traffic encrypted with post-quantum cryptographic protocols.
 
-This docker image contains a version of [wireshark](https://www.wireshark.org/) built to also properly display quantum-safe crypto (QSC) TLS operations.
+## Quick Start Guide
 
-To this end, it contains references to algorithms supported by [liboqs](https://github.com/open-quantum-safe/liboqs) and [OQS-OpenSSL](https://github.com/open-quantum-safe/openssl) from the [OpenQuantumSafe](https://openquantumsafe.org) project.
+```bash
```

Comment:
This is build documentation that should not be necessary (read: is probably confusing) for normal users (of a readily built dockerimage) and thus should only be present in the README.md.